Sem. 04/Pract. 04/Task01.cpp

```diff
@@ -9,11 +9,11 @@ int strLength(const char* str) {
         length++;
     }
 
-    return length;
+    return length + 1;
```

Comment:
I meant the for loop, lenght + 1 is worse. Null terminator is not included in strlen!